site/content/in-dev/unreleased/quickstart.md

```diff
@@ -51,6 +51,34 @@ If you plan to deploy Polaris inside [Docker](https://www.docker.com/), you'll n
 brew install --cask docker
 ```
 
+If you encounter Docker permission-related issues, you can refer to the link [gradle](https://github.com/apache/polaris/pull/971)
+
+you may encounter issues related to seccomp configuration. It is recommended to set the seccomp profile to "unconfined" to avoid potential permission problems.
+
+To set the seccomp profile to "unconfined" for a single container, you can use the following command when running the container
```

Comment:
```suggestion
There could be a [Docker permission issues](https://github.com/apache/polaris/pull/971) related to seccomp configuration. Set the seccomp profile to "unconfined" for a single container when running the container:
```

Reply:
Oh, I see -- this can happen with any distro that uses seccmp. Have we tested this flow with e.g. CentOS?